apps/condo/domains/contact/hooks/useTableColumns.tsx

```diff
@@ -15,10 +15,20 @@ export const useTableColumns = (sort: Array<string>, filters: IFilters,
     const PhoneMessage =  intl.formatMessage({ id: 'Phone' })
     const EmailMessage = intl.formatMessage({ id: 'field.EMail' })
     const AddressMessage = intl.formatMessage({ id: 'pages.condo.property.field.Address' })
+    const ShortFlatNumber = intl.formatMessage({ id: 'field.FlatNumber' })
 
     const sorterMap = createSorterMap(sort)
     const search = getFilteredValue(filters, 'search')
 
+    const renderAddress = (address, record) => {
```

Comment:
Seems like we use this render not only in contact table, but in every single table...Some time ago I've created common renders in `@condo/domains/common/components/Table/Renders`. Seems it's locally duplicates it. Please use common render for address or modify it for all tables if you need to.

Reply:
done